recipes-wpe/wpeframework/include/avs.inc

```diff
@@ -0,0 +1,48 @@
+# Provides configuration for the AVS Plugin
+WPE_AVS_AUTOSTART ??= "true"
+WPE_AVS_NAME ??= ""
+WPE_AVS_PLATFORM ??= ""
+WPE_AVS_DATA_PATH ??= "/usr/share/WPEFramework/AVS"
+WPE_AVS_PERSISTENT_PATH ??= "/home/root/AVS"
+WPE_AVS_ALEXA_CLIENT_CONFIG ??= "${WPE_AVS_DATA_PATH}/AlexaClientSDKConfig.json"
+
+# Possible Log Levels: NONE/CRITICAL/ERROR/WARN/DEBUG0/DEBUG9
+WPE_AVS_LOG_LEVEL ??= "CRITICAL"
+
+PACKAGECONFIG[avsclient] = "\
+    -DPLUGIN_AVS=ON \
+    -DPLUGIN_AVS_AUTOSTART="${WPE_AVS_AUTOSTART}" \
+    -DPLUGIN_AVS_NAME="${WPE_AVS_NAME}" \
+    -DPLUGIN_AVS_PLATFORM="${WPE_AVS_PLATFORM}" \
+    -DPLUGIN_AVS_PERSISTENT_PATH="${WPE_AVS_PERSISTENT_PATH}" \
+    -DPLUGIN_AVS_DATA_PATH="${WPE_AVS_DATA_PATH}" \
+    -DPLUGIN_AVS_ALEXA_CLIENT_CONFIG="${WPE_AVS_ALEXA_CLIENT_CONFIG}" \
+    -DPLUGIN_AVS_LOG_LEVEL="${WPE_AVS_LOG_LEVEL}" \
+    ,-DPLUGIN_AVS=OFF,avs-device-sdk \
+"
+
+WPE_AVS_SMART_SCREEN_CONFIG ??= "${WPE_AVS_DATA_PATH}/SmartScreenSDKConfig.json"
+WPE_AVS_ENABLE_SMART_SCREEN ??= "false"
+PACKAGECONFIG[avsclient_smartscreen] = "\
+    -DPLUGIN_AVS_ENABLE_SMART_SCREEN_SUPPORT=ON \
+    -DPLUGIN_AVS_SMART_SCREEN_CONFIG="${WPE_AVS_SMART_SCREEN_CONFIG}" \
+    -DPLUGIN_AVS_ENABLE_SMART_SCREEN="${WPE_AVS_ENABLE_SMART_SCREEN}" \
+    ,-DPLUGIN_AVS_ENABLE_SMART_SCREEN_SUPPORT=OFF,alexa-smart-screen-sdk \
+"
+
+PACKAGECONFIG[avsclient_portaudio] = "\
+    -DPLUGIN_AVS_AUDIOSOURCE="PORTAUDIO" \
+    ,,portaudio-v19,portaudio-v19 \
+"
+
+WPE_AVS_ENABLE_KWD ??= "true"
```

Comment:
Do we really want to build the KWD by default? KWD is closed source component not available to everyone, so some clients might use the functionality without KWD

Reply:
moved this to meta-wpe-restricted and adding using bbappend